frontend/src/pages/Configure/Storage.js

```diff
@@ -448,31 +449,51 @@ function EbsSettings({index}) {
   )
 }
 
+/*
+ * Used to configure the Storage part of the Cluster wizard:
+ *  - controls whether a storage type can be created or just linked
+ *  - specify if a storage type can be mounted as a file system or just one of its volumes
+ */
+const STORAGE_TYPE_PROPS = {
```

Comment:
Since we have introduced this, we should use `STORAGE_TYPE_PROPS[storageType].canCreate` in [Filesystems.js](https://github.com/aws-samples/pcluster-manager/blob/main/frontend/src/pages/Clusters/Filesystems.js#L31-L32) as well.

Reply:
We'll keep these kind of refactoring for the near future (when converting to Typescript) when we can then make storage types really "pluggable"